sphinxext/tabs.py

```diff
@@ -61,7 +61,7 @@
      {{ end }}
      <div class="tabs__content" role="tabpanel">
        {{ for tab in tabs %FILTER% }}
-       <div class="tabpanel-{{ tab.id asIdentifier }}">
+       <div class="tabpanel-{{ tab.id asIdentifier }}" data-tabid="{{ tab.id asIdentifier }}">
```

Comment:
`data-*` is the HTML5 data attribute convention. Attributes beginning with `data-` are reserved for applications, and are used by `HTMLElement.dataset`. As for `data-tabId` vs `data-tab-id`... 🤷‍♂️

The `data-tabid` on line 44 is necessary because we still use the tab strip element tabId properties in the `TabSet.setup()` method.